src/components/Lend/LoanSearch/LoanSearchFilterChips.vue

```diff
@@ -82,30 +82,93 @@ export default {
 			default: () => {}
 		}
 	},
-	data() {
-		return {
-			items: []
-		};
-	},
-	methods: {
-		getLabelsFromState() {
+	computed: {
+		items() {
+			return this.getLabelsFromState(this.loanSearchState, this.allFacets);
 		}
 	},
-	mounted() {
-		// this.items = [...this.loanSearchState.gender];
-		console.log(this.loanSearchState);
-		console.log(this.allFacets);
-	},
-	watch: {
-		loanSearchState(newVal) {
-			console.log(newVal.gender);
-			if (newVal) {
-				this.items.gender = newVal;
+	methods: {
+		formatRemovedFacet(facetType, facet) {
+			switch (facetType) {
+				case 'Gender':
+					return {
+						gender: null
+					};
+				case 'Sector':
+					return {
+						sectorId: [facet.id]
```

Comment:
I tested these changes locally today, and this line isn't removing the `sectorId` filter. The bug is logged here: https://kiva.atlassian.net/browse/VUE-1099